app/src/app/components/stix/stix-list/stix-list.component.html

```diff
@@ -13,7 +13,7 @@
                     </mat-form-field>
         
                     <!-- don't show options for collections or if loading data not from REST API -->
-                    <mat-form-field appearance="fill" *ngIf="config.showFilters && !(config.hasOwnProperty('stixObjects'))">
+                    <mat-form-field appearance="fill" *ngIf="config.showFilters">
```

Comment:
This inadvertently displays filters on `stix-lists` across Workbench in places where it may not make sense.  The `AddDialogComponent` should be updated to disable the `stix-list` filters, since the lists are already filtered in this view. For example: the list of available tactics while editing a technique (already filtered by non-revoked/non-deprecated/domain) and default marking definitions (which do not support state/status).

<img width="1102" alt="image" src="https://github.com/center-for-threat-informed-defense/attack-workbench-frontend/assets/48832936/9aae6607-7ea6-4a2a-be85-b4b17ceed705">